### ISSUE TITLE
4.08 support

### ISSUE THREAD

**Issue by copy** [ocaml-ppx/ppx_tools#69]
On a 4.08.0+beta1 switch:

```
File "ast_convenience.ml", line 1:
Error: The implementation ast_convenience.ml
       does not match the interface ast_convenience.cmi:
       Values do not match:
         val has_attr : 'a -> ('a Location.loc * 'b) list -> bool
       is not included in
         val has_attr : string -> Parsetree.attributes -> bool
       File "ast_convenience.mli", line 108, characters 0-42:
         Expected declaration
       File "ast_convenience.ml", line 121, characters 4-12:
         Actual declaration
```

**Comment by jhwoodyatt** [comment]
I believe this is addressed by #70.

**Comment by copy** [comment]
The master branch installs fine on 4.08. Closing.
